### PR TITLE
PR: Profile Screen fix

### PR DESCRIPTION
**Pull request by JoniDutoit**
## Overview

This small PR aims to fix the behavior and UI display of the Profile Screen after the refactor. Merge this PR will give the following results as shown below

## Results
Before | After
---------- | -----------
 |

### CODE REVIEW

File: app/src/main/java/com/lastaoutdoor/lasta/ui/screen/profile/ProfileScreen.kt [hunk 1]
```diff
@@ -429,89 +435,99 @@ fun Chart(activities: List<UserActivity>, timeFrame: TimeFrame, sport: ActivityT
...
-                          text =
-                              String.format(
-                                  "%.2f", metersToKilometers(climb.totalElevation.toLong())),
-                          fontWeight = FontWeight.Bold,
-                          modifier = Modifier.testTag("TestAndrew1"))
-                      Text(text = LocalContext.current.getString(R.string.elevation))
-                    }
-                    ActivityType.BIKING -> {}
+  Text(
+      "Recent Activities",
```
Comment (Andy130604) [inline app/src/main/java/com/lastaoutdoor/lasta/ui/screen/profile/ProfileScreen.kt:440]: Should have this in string resources

**Review by Andy130604 — CHANGES_REQUESTED** [review]
Everything looks good you have one small fix to do: Have recent activities text be in string resources.

File: app/src/main/java/com/lastaoutdoor/lasta/ui/screen/profile/ProfileScreen.kt (hunk 1 above)
Comment (JoniDutoit) [reply app/src/main/java/com/lastaoutdoor/lasta/ui/screen/profile/ProfileScreen.kt]: Thank you for pointing it out, indeed I didn't see it ! Made the change rn

**Review by Andy130604 — APPROVED** [review]
Everything seems fixed!

**Pull request merged**